torchrl/data/tensordict/utils.py

```diff
@@ -82,3 +82,52 @@ def _getitem_batch_size(
     list_iter_bs = list(iter_bs)
     bs += list_iter_bs
     return torch.Size(bs)
+
+
+def convert_ellipsis_to_idx(idx: tuple, batch_size: list):
+    """
+    Given an index containing an ellipsis or just an ellipsis, converts any ellipsis to slice(None)
+    Example: idx = (..., 0), batch_size = [1,2,3] -> new_index = (slice(None), slice(None), 0)
+
+    Args:
+        idx: Input index
```

Comment:
perhaps add dtype? `idx (tuple): ...` `batch_size (list): ...`

Reply:
do you mean add typing to the variables under "Args"?

Reply:
Yes 
The doc should say what type you expect for the inputs.
IMO that is more important than type hinting

Reply:
Changed to Tuple and List from typing library